server/src/main/java/com/senlainc/warsaw/tyurin/util/HibernateUtil.java

```diff
@@ -0,0 +1,50 @@
+package com.senlainc.warsaw.tyurin.util;
+
+import com.senlainc.warsaw.tyurin.annotation.DependencyClass;
+import com.senlainc.warsaw.tyurin.annotation.DependencyInitMethod;
+import com.senlainc.warsaw.tyurin.entity.Craftsman;
+import com.senlainc.warsaw.tyurin.entity.GaragePlace;
+import com.senlainc.warsaw.tyurin.entity.Order;
+import org.apache.log4j.Logger;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+
+@DependencyClass
+public class HibernateUtil {
+
+    private final static Logger logger = Logger.getLogger(HibernateUtil.class);
+    private static HibernateUtil INSTANCE;
+    private SessionFactory sessionFactory;
+
+    public static HibernateUtil getInstance() {
```

Comment:
Also no need to have instance now. After you created DI container you now will have same copy of object if you inject it properly

Reply:
also check other services on this